upup/pkg/fi/cloudup/awsup/machine_types.go

```diff
@@ -80,6 +80,15 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
 	// NOTE: Content below is auto generated by `make update-machine-types`
 	// BEGIN GENERATED CONTENT
 
+	// WARNING: ENIs not known for i3en.metal
+	// WARNING: IPs per ENI not known for i3en.metal
+	// WARNING: ENIs not known for c5.24xlarge
+	// WARNING: IPs per ENI not known for c5.24xlarge
+	// WARNING: ENIs not known for c5.12xlarge
+	// WARNING: IPs per ENI not known for c5.12xlarge
+	// WARNING: ENIs not known for c5.metal
+	// WARNING: IPs per ENI not known for c5.metal
```

Comment:
In order to remove these warnings, please manually add the required data here: https://github.com/kubernetes/kops/blob/master/hack/machine_types/vpc_ip_resource_limit.go

Above the two sections you see links to where you should be able to get them!

Reply:
Then rerun the make command to regenerate. Thanks!

Reply:
thanks for that, https://docs.aws.amazon.com/en_us/AWSEC2/latest/UserGuide/using-eni.html is not updated yet, I'm trying to get this through a contact at AWS.

Reply:
@mikesplain done but now i3en.metal is in the list but number of max ENI and max IP per ENI is not available for this instance type :)